app/views/layouts/_header.html.erb

```diff
@@ -43,5 +43,5 @@
 <body class="<%= controller_name %> <%= action_name %>">
   <%=render partial: 'layouts/topnav'%>
   
-  <div class="container-fluid flex-grow-1">
+  <div class="flex-grow-1">
```

Comment:
This will not for all the pages.

Container fluid here is doing left and right padding on all the pages on the website; removing it here may work for your case (the home page) but probably not for the other pages (e.g account or summary pages)

Reply:
In this case, I don't think that it's a good idea to put it in the container that wrap all pages, it's better to add it just in the needed pages.

Reply:
But do you not think, that all the pages have this padding, and  that's why they did